src/std/io/interface.ss

```diff
@@ -54,7 +54,8 @@
   (delimit (limit :~ nonnegative-fixnum?))
 
   ;; resets the underlying reader and buffer state, allowing reuse of buffers.
-  (reset! (reader : Reader)))
+  (reset! (reader : Reader) (close? :~ boolean? := #t)))
```

Comment:
No need for contract check here, any truthy value will do...

Reply:
So just `(close? := #t)` ?

Reply:
yup.